### PR TITLE
create subscription and reviews model

### PR DESCRIPTION
**Pull request by rawaneyad**
## Create subscription and reviews model
Tables
db

- [x] Create subscription model
- [x] Create reviews model
- [x] Update index

### CODE REVIEW

File: server/database/models/subscription.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/connection';
+
+const Subscription = sequelize.define('Subscription', {
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  created_at: {
```
Comment (AhmedQeshta) [inline server/database/models/subscription.ts:10]: Must Remove  create_at, because sequelize create it by default

**Review by AhmedQeshta — CHANGES_REQUESTED** [review]

File: server/database/models/subscription.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/connection';
+
+const Subscription = sequelize.define('Subscription', {
```
Comment (LinaYahya) [inline server/database/models/subscription.ts:4]:
```suggestion
const Subscription = sequelize.define('subscriptions', {
```

File: server/database/models/subscription.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  type: {
+    type: DataTypes.STRING,
+    allowNull: false,
```
Comment (LinaYahya) [inline server/database/models/subscription.ts:12]:
```suggestion
    allowNull: false,
     values: ['one', 'six'],
```
I would prefer to specify values either to be six for six months subscription or one

File: server/database/models/reviews.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/connection';
+
+const Reviews = sequelize.define('Reviews', {
```
Comment (LinaYahya) [inline server/database/models/reviews.ts:4]:
```suggestion
const Reviews = sequelize.define('reviews', {
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by AhmedQeshta — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**